src/components/Dropdown/asyncDropdown/AsyncDropdown.jsx

```diff
@@ -21,7 +21,7 @@ class AsyncDropDown extends React.Component {
     super(props);
     this.state = {
       items: [],
-      inputValue: '',
+      inputValue: 'podkast om',
```

Comment:
Antar dette er en typo.
```suggestion
      inputValue: '',
```

Reply:
Er debugging, har ikke fullført PR'en ennå, etter audio-api oppdateringene :)